Arch docs missed requirement

# Bug report

## To Reproduce

```
pahaz@PahazMac16 my-keystone-arch-docs % yarn 
yarn install v1.22.4
[1/4] 🔍  Resolving packages...
warning @arch-ui/tooltip > react-popper > popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
warning gatsby > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning gatsby > @babel/polyfill > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning gatsby > gatsby-cli > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning gatsby > gatsby-cli > yurnalist > babel-runtime > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
warning gatsby > micromatch > snapdragon > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
warning gatsby > micromatch > snapdragon > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning "@arch-ui/button > react-pseudo-state@2.2.2" has unmet peer dependency "react-dom@^16.3.0".
warning "@arch-ui/controls > react-radios@1.1.0" has unmet peer dependency "react-dom@^16.3.0".
warning "@arch-ui/controls > react-radios > unstated > create-react-context@0.1.6" has unmet peer dependency "prop-types@^15.0.0".
warning " > @arch-ui/dialog@0.0.19" has unmet peer dependency "react-dom@^16.13.0".
warning "@arch-ui/dialog > focus-trap-react@6.0.0" has unmet peer dependency "react-dom@0.14.x || ^15.0.0 || ^16.0.0".
warning "@arch-ui/dialog > react-scrolllock > react-node-resolver@2.0.1" has unmet peer dependency "react-dom@^15.0 || ^16.0".
warning "@arch-ui/dialog > @arch-ui/modal-utils > react-transition-group@4.3.0" has unmet peer dependency "react-dom@>=16.6.0".
warning " > @arch-ui/dropdown@0.0.16" has unmet peer dependency "react-dom@^16.13.0".
warning " > @arch-ui/popout@0.0.17" has unmet peer dependency "react-dom@^16.13.0".
warning "@arch-ui/select > react-select@3.1.0" has unmet peer dependency "react-dom@^16.8.0".
warning " > @arch-ui/tooltip@0.1.10" has unmet peer dependency "react-dom@^16.13.0".
warning " > @reach/router@1.3.3" has unmet peer dependency "react-dom@15.x || 16.x || 16.4.0-alpha.0911da3".
warning " > gatsby@2.20.23" has unmet peer dependency "react-dom@^16.4.2".
warning "gatsby > gatsby-react-router-scroll@2.2.1" has unmet peer dependency "react-dom@^0.14.0 || ^15.0.0 || ^16.0.0".
warning "gatsby > gatsby-link@2.3.3" has unmet peer dependency "react-dom@^16.4.2".
warning "gatsby > react-hot-loader@4.12.20" has unmet peer dependency "@types/react@^15.0.0 || ^16.0.0".
warning "gatsby > react-hot-loader@4.12.20" has unmet peer dependency "react-dom@^15.0.0 || ^16.0.0".
warning "gatsby > @typescript-eslint/eslint-plugin > tsutils@3.17.1" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta".
warning "gatsby > gatsby-recipes > ink-box@1.0.0" has unmet peer dependency "ink@>=2.0.0".
warning "gatsby > gatsby-recipes > ink-link@1.1.0" has unmet peer dependency "ink@>=2.0.0".
warning "gatsby > gatsby-recipes > ink-select-input@3.1.2" has unmet peer dependency "ink@^2.0.0".
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
✨  Done in 73.77s.
pahaz@PahazMac16 my-keystone-arch-docs % ls
CHANGELOG.md    babel.config.js netlify.toml    node_modules    package.json    src             yarn.lock
pahaz@PahazMac16 my-keystone-arch-docs % s .
pahaz@PahazMac16 my-keystone-arch-docs % yarn start
yarn run v1.22.4
$ gatsby develop
╔════════════════════════════════════════════════════════════════════════╗
║                                                                        ║
║   Gatsby collects anonymous usage analytics                            ║
║   to help improve Gatsby for all users.                                ║
║                                                                        ║
║   If you'd like to opt-out, you can use `gatsby telemetry --disable`   ║
║   To learn more, checkout https://gatsby.dev/telemetry                 ║
║                                                                        ║
╚════════════════════════════════════════════════════════════════════════╝
success open and validate gatsby-configs - 0.012s
success load plugins - 0.085s
success onPreInit - 0.006s
success initialize cache - 0.008s
success copy gatsby files - 0.034s
success onPreBootstrap - 0.009s
success createSchemaCustomization - 0.004s
success source and transform nodes - 0.014s
success building schema - 0.119s
success createPages - 0.002s
success createPagesStatefully - 0.045s
success onPreExtractQueries - 0.001s
success update schema - 0.019s
success extract queries from components - 0.071s
success write out requires - 0.007s
success write out redirect data - 0.002s
success onPostBootstrap - 0.002s
⠀
info bootstrap finished - 2.890 s
⠀
success run queries - 0.036s - 4/4 110.35/s

 ERROR 

There was an error compiling the html.js component for the development server.

See our docs page on debugging HTML builds for help https://gatsby.dev/debug-html  Cannot find module 'react-dom/server'


  174 |     function checkType(isRequired, props, propName, componentName, location, propFullName, secret) {
  175 |       componentName = componentName || ANONYMOUS;
> 176 |       propFullName = propFullName || propName;
      | ^
  177 | 
  178 |       if (secret !== ReactPropTypesSecret) {
  179 |         if (throwOnDirectAccess) {


  WebpackError: Cannot find module 'react-dom/server'
  
  - factoryWithTypeCheckers.js:176 Function.Module._resolveFilename
    node_modules/prop-types/factoryWithTypeCheckers.js:176:1
  
  - factoryWithTypeCheckers.js:58 Function.Module._load
    node_modules/prop-types/factoryWithTypeCheckers.js:58:1
  
  - factoryWithTypeCheckers.js:238 Module.require
    node_modules/prop-types/factoryWithTypeCheckers.js:238:1
  
  - factoryWithTypeCheckers.js:352 Module._compile
    node_modules/prop-types/factoryWithTypeCheckers.js:352:1
  
  - factoryWithTypeCheckers.js:372 Object.Module._extensions..js
    node_modules/prop-types/factoryWithTypeCheckers.js:372:1
  
  - factoryWithTypeCheckers.js:196 Module.load
    node_modules/prop-types/factoryWithTypeCheckers.js:196:1
  
  - factoryWithTypeCheckers.js:95 Function.Module._load
    node_modules/prop-types/factoryWithTypeCheckers.js:95:1
  
  - factoryWithTypeCheckers.js:238 Module.require
    node_modules/prop-types/factoryWithTypeCheckers.js:238:1
  
  - factoryWithTypeCheckers.js:72 emit
    node_modules/prop-types/factoryWithTypeCheckers.js:72:1
  



———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
4 pages                                                                                                             Failed                                                                                                            @arch-ui/docs
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

## Workaround

How to fix it!

```
pahaz@PahazMac16 my-keystone-arch-docs % yarn add react-dom       
yarn add v1.22.4
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning "@arch-ui/controls > react-radios > unstated > create-react-context@0.1.6" has unmet peer dependency "prop-types@^15.0.0".
warning "gatsby > react-hot-loader@4.12.20" has unmet peer dependency "@types/react@^15.0.0 || ^16.0.0".
warning "gatsby > @typescript-eslint/eslint-plugin > tsutils@3.17.1" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta".
warning "gatsby > gatsby-recipes > ink-box@1.0.0" has unmet peer dependency "ink@>=2.0.0".
warning "gatsby > gatsby-recipes > ink-link@1.1.0" has unmet peer dependency "ink@>=2.0.0".
warning "gatsby > gatsby-recipes > ink-select-input@3.1.2" has unmet peer dependency "ink@^2.0.0".
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
success Saved 1 new dependency.
info Direct dependencies
└─ react-dom@16.13.1
info All dependencies
└─ react-dom@16.13.1
✨  Done in 4.54s.
pahaz@PahazMac16 my-keystone-arch-docs % yarn start               
yarn run v1.22.4
$ gatsby develop
success open and validate gatsby-configs - 0.008s
success load plugins - 0.021s
success onPreInit - 0.005s
info One or more of your plugins have changed since the last time you ran Gatsby. As
a precaution, we're deleting your site's cache to ensure there's no stale data.
success initialize cache - 0.019s
success copy gatsby files - 0.033s
success onPreBootstrap - 0.007s
success createSchemaCustomization - 0.004s
success source and transform nodes - 0.013s
success building schema - 0.107s
success createPages - 0.001s
success createPagesStatefully - 0.045s
success onPreExtractQueries - 0.001s
success update schema - 0.018s
success extract queries from components - 0.101s
success write out requires - 0.007s
success write out redirect data - 0.001s
success onPostBootstrap - 0.002s
⠀
info bootstrap finished - 2.014 s
⠀
success run queries - 0.047s - 4/4 84.95/s
⠀
You can now view @arch-ui/docs in the browser.
⠀
  http://localhost:8000/
⠀
View GraphiQL, an in-browser IDE, to explore your site's data and schema
⠀
  http://localhost:8000/___graphql
⠀
Note that the development build is not optimized.
To create a production build, use gatsby build
⠀
success Building development bundle - 2.388s
success Re-building development bundle - 0.051s


———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
4 pages                                                                                                            Success                                                                                                            @arch-ui/docs
```

--- a/packages/arch/README.md
+++ b/packages/arch/README.md
@@ -9,7 +9,7 @@ Resources, tooling, and design guidelines by KeystoneJS using [GastbyJS](https:/
 
 ## Getting Started
 
-To start, run
+To start, run:
 
 ```sh
 cd docs

--- a/packages/arch/docs/package.json
+++ b/packages/arch/docs/package.json
@@ -29,7 +29,8 @@
     "@reach/router": "^1.3.3",
     "clipboard-copy": "^3.0.0",
     "gatsby": "^2.13.25",
-    "react": "^16.13.0"
+    "react": "^16.13.0",
+    "react-dom": "^16.13.0"
   },
   "scripts": {
     "start": "gatsby develop",

